changelog.md

```diff
@@ -1,3 +1,10 @@
+### <a name="2.14.0"></a> 2.14.0
+
+* Add PopulateIndexMetrics to FeedOptions
+* Add backend request duration in milliseconds (BELatencyMs) to RequestDiagnosticsString
+* Improved allocations on Direct/TCP
+* Improved latency for Gateway mode users on .NET Framework by disabling Nagle algorithm
+
```

Comment:
* Fixed race condition in direct + tcp mode causing SDK generated internal server errors and invalid operation exceptions 
* Fixed race condition in direct + tcp mode causing unncessary connections to be created by concurrent requests